### ISSUE TITLE
Offline access

### ISSUE THREAD

**Issue by cossio** [CellarD0-0r/whatever#19]
Can we have offline access to evernote with this app?

It would be nice that I could take a note, or modify existing notes, when the PC is offline, and then when it gets online, the note is synced automatically.

**Comment by stuartluscombe** [comment]
I'll second this request!

**Comment by dddobriak** [comment]
I think this app like iframe and can't offline work

**Comment by ketanhwr** [comment]
Yeah, this application just opens the official evernote website in an [Electron](https://electron.atom.io/) contained window. Had it used the Electron API to render it's own frontend, something like offline support would've been added (the former would obviously take a lot of work, and the latter is something of a mini-project itself).
